gallery/src/components/NxTag/NxSelectableTagExample.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+import { NxSelectableTag } from '@sonatype/react-shared-components';
+
+function NxSelectableTagExample() {
+  const [tagSelected1, setTagSelected1] = useState(false),
```

Comment:
A more typical way to do this would be with a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) that stores all the currently active tags.

Reply:
> A more typical way to do this would be with a Set that stores all the currently active tags.

https://github.com/sonatype/sonatype-react-shared-components/pull/295